src/eth/index.ts

```diff
@@ -1,11 +1,28 @@
 import { TransactionConfig, PastLogsOptions } from 'web3-core';
 import { RPC_CALLS } from '../eth/constants';
-import { RpcCallNames, RequestArgs, RPCPayload, SentSDKMessage, Communicator } from './../types';
-import { SDK_MESSAGES } from './../communication/messageIds';
+import {
+  RpcCallNames,
+  Communicator,
+  Log,
+  BlockTransactionString,
+  BlockTransactionObject,
+  Web3TransactionObject,
+  RPCPayload,
+} from '../types';
+import { METHODS } from '../communication/methods';
 
-const inputFormatters = {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Formatter = (arg: any) => any;
+
+const inputFormatters: Record<string, Formatter> = {
   defaultBlockParam: (arg = 'latest') => arg,
-  fullTxObjectParam: (arg = false) => arg,
+  fullTxObjectParam: (arg = false): boolean => arg,
```

Comment:
`fullTxObject` is a `boolean`?

Reply:
yes, see `getBlockByHash` params there: https://eth.wiki/json-rpc/API

they don't have hyperlinks for calls :/

Reply:
ahhh it is the params if the full transactions should be returned when requesting block info ... ok kind of confusing, maybe a comment would be nice :)